tools/stern/src/Stern/API.hs

```diff
@@ -480,7 +479,7 @@ routes = do
   document "GET" "getConsentLog" $ do
     summary "Fetch the consent log given an email address of a non-user"
     notes "Relevant only internally at Wire"
-    Doc.parameter Doc.Query "email" Doc.string' $ do
```

Comment:
i like redundant `do`.  also it guides ormolu in the right direction sometimes, but i forgot where.  can we ignore this as well?